synthtool/gcp/gapic_microgenerator.py

```diff
@@ -134,6 +134,14 @@ def _generate_code(
         # The time has come, the walrus said, to talk of actually running
         # the code generator.
         sep = os.path.sep
+
+        # try to figure out user ID and stay compatible.
+        # If there is no `os.getuid()`, fallback to `getpass.getuser()`
+        try:
+            user = str(os.getuid())
+        except:
```

Comment:
```suggestion
        except AttributeError:
```

Reply:
Thanks! Fixed. This was also flagged by `flake8`.